ut-user/index.js

```diff
@@ -16,12 +20,28 @@ function getCrypt(cryptKey) {
     return crypt;
 }
 
+function getIPAddress() {
+  var interfaces = os.networkInterfaces();
+  for (var devName in interfaces) {
+    var iface = interfaces[devName];
+
+    for (var i = 0; i < iface.length; i++) {
+      var alias = iface[i];
+      if (alias.family === 'IPv4' && alias.address !== '127.0.0.1' && !alias.internal)
+        return alias.address;
+    }
+  }
+  return '0.0.0.0';
+}
+
 module.exports = {
     init: function(b) {
         getCrypt(b.config.masterCryptKey);
         importMethod = b.importMethod.bind(b);
         checkMethod = b.config['identity.check'];
         debug = b.config.debug;
+        applicationVersion = b.config.applicationVersion;
```

Comment:
" On the other hand, if we have a way to get the version of implementation project from the other modules , then that would be a better solution over this " 
Hi Kanagu Anna, This is possible. I have made the changes and committed it. we can get the implementation project version from the following code changes.
var path = require('path');
var packageObj = require(path.join(path.dirname(require.main.filename), 'package.json'));
 Please correct me If it is wrong